### ISSUE TITLE
Could not start IPFS instance

### ISSUE THREAD

**Issue by Vivi728** [aragon/aragon-desktop#72]
when I run `npm start` , I got this： 
![image](https://user-images.githubusercontent.com/13521173/59194152-a7636e00-8bba-11e9-9c12-baca9367019c.png)
How to start IPFS instance？

**Comment by luisivan** [comment]
This is basically the same issue as https://github.com/aragon/aragon-cli/issues/505, one link seems to be broken in one of the dependencies we are using :/

**Comment by Vivi728** [comment]
That is to say, there is no solution yet，right?

**Comment by luisivan** [comment]
Not sure, any thoughts @0xGabi?

**Comment by 0xGabi** [comment]
@Vivi728 can you check again? We experienced an issue with the reliability when downloading IPFS, we will work to fix it. In case you are still experience the same you can try running `aragon ipfs` before ran `npm start`. To install the **aragon** binary ran `npm i -g @aragon/cli`

**Comment by Vivi728** [comment]
It'is so bad when I run `npm i -g @aragon/cli`
![image](https://user-images.githubusercontent.com/13521173/59401722-3ac0bd00-8dce-11e9-976c-988c0f13cbdb.png)

**Comment by 0xGabi** [comment]
Hey @Vivi728 I believe this is a problem with how you have permissions setup for node on you machine. Can you try: `sudo npm i -g --unsafe-perm @aragon/cli`

**Comment by Vivi728** [comment]
![image](https://user-images.githubusercontent.com/13521173/59479007-f98fe200-8e8d-11e9-9d69-6047d0e58900.png)

**Comment by 0xGabi** [comment]
@Vivi728 sorry for all this troubles. This is an issue with IPFS the last couple of days. We already ask them for help here: https://github.com/ipfs/go-ipfs/issues/6451

**Comment by Vivi728** [comment]
@0xGabi  Is there any progress?

**Comment by 0xGabi** [comment]
@Vivi728 seems to be working now. But might be unreliable. We are considering other options for future versions.

**Comment by Vivi728** [comment]
@0xGabi It seems that everything is back to its original point.
![image](https://user-images.githubusercontent.com/13521173/59583246-46cdb700-910d-11e9-81be-010da159e120.png)

**Comment by 0xGabi** [comment]
Hey sorry that I took longer to answer you. I think you still don't have ipfs installed, [this answer might help](https://github.com/aragon/aragon-cli/issues/505#issuecomment-502852557) if you struggle with that.

Then I also recommend you ran `aragon ipfs` before `npm run start`. I tested both linux and macOS and it worked that way for me. 

Anyways I bumped a couple of issue with openssl and scrypt dependency. I plan to take a look more in detail and update the repo a bit..hopefully improving the developer experience.

**Comment by Vivi728** [comment]
@0xGabi  Hey is there a complete installation documentation?

**Comment by 0xGabi** [comment]
@Vivi728 You mean for Aragon desktop?

**Comment by Vivi728** [comment]
and  aragon-cli .

**Comment by Vivi728** [comment]
@0xGabi I've install Aragon desktop successfully，but something wrong with aragon cli like that:
![image](https://user-images.githubusercontent.com/13521173/60065789-a5131f00-9737-11e9-9116-5719c4bd4f27.png)
Is thak ok？

**Comment by 0xGabi** [comment]
Hey you should ran: `npm run prepare` instead. We explain why we are doing it as well as other development decisions on our [internal documentation.](https://github.com/aragon/aragon-cli/tree/master/docs-internal) Hope you find it useful.

**Comment by Vivi728** [comment]
@0xGabi  Yeah, but I need to start ipfs service manually , then start the Aragon desktop, otherwise I will get an error like " Could not start IPFS service" , just like before.

**Comment by 0xGabi** [comment]
Ok, thanks for clarifying it
